circuits/circom/elGamalEncryption.circom

```diff
@@ -0,0 +1,107 @@
+pragma circom 2.0.0;
+// ElGamal Encryption
+include "../node_modules/circomlib/circuits/escalarmulany.circom";
+include "../node_modules/circomlib/circuits/babyjub.circom";
+include "../node_modules/circomlib/circuits/bitify.circom";
+include "../node_modules/circomlib/circuits/mux1.circom";
+
+// 0 --> (0, 1); 1 --> BASE point
+template BitToPoint() {
+    signal input bit;
+    signal output M[2];
+
+    component pointSelector = MultiMux1(2);
+    pointSelector.s <== bit;
+
+    pointSelector.c[0][0] <== 0;
+    pointSelector.c[0][1] <== 5299619240641551281634865583518297030282874472190772894086521144482721001553;
+    pointSelector.c[1][0] <== 1;
+    pointSelector.c[1][1] <== 16950150798460657717958625567821834550301663161624707787222815936182638968203;
```

Comment:
could you please link to where you got the points from?

Reply:
verifySignature.circom:74

` // Calculate left side of equation left = S*B8
    var BASE8[2] = [
        5299619240641551281634865583518297030282874472190772894086521144482721001553,
        16950150798460657717958625567821834550301663161624707787222815936182638968203
    ];
    `